### PR TITLE
141 miscellaneous refactor and restyle

### PR DESCRIPTION
**Pull request by hepnerthomas**
Trello:
- https://trello.com/c/T9zos4Qp/141-miscellanous-refactor-and-restyle-ratings-reviews
- https://trello.com/c/iPAjpvHs/93-complete-review-list

**Notes:**
- created folders for files
image
- setState for `productId` in `App.jsx` always sets to a number
- style: resize text on product breakdown: 
image
- refactor: <a> to <button> tags: 
image
- bug fix: recommendations
image

### CODE REVIEW

File: client/src/components/QA/Questions/AnswerModal.jsx [hunk 1]
```diff
@@ -91,16 +91,17 @@ class AnswerModal extends React.Component {
...
-    axiosConfig.post(`/qa/questions/${question_id}/answers`, {
-      body, name, email, photos: imgFileURLs,
-    })
-      .then(() => {
-        hideModal();
-        getAnswers();
-      })
-      .catch((err) => {
-        console.log(err);
-      });
```
Comment (amberlyM) [inline client/src/components/QA/Questions/AnswerModal.jsx]: Would you undo this stuff? Thanks for your help earlier

File: client/src/components/App/App.jsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ class App extends React.Component {
 
   updateProductId(id) {
     this.setState({
-      productId: id,
+      productId: Number(id),
```
Comment (amberlyM) [inline client/src/components/App/App.jsx:26]: great idea!

File: client/src/components/RatingsReviews/Breakdowns/ProductBreakdown.module.css [hunk 3]
```diff
@@ -14,10 +14,16 @@ h2 {
   display: flex;
   flex-direction: column;
   /* padding: 0.5em 0.5em; */
-  font-weight: bold;
 
 }
 
+.characteristicText {
```
Comment (amberlyM) [inline client/src/components/RatingsReviews/Breakdowns/ProductBreakdown.module.css:20]: Screen Shot 2022-09-08 at 3 14 22 PM

For some reason I still see this

**Review by amberlyM — CHANGES_REQUESTED** [review]

File: client/src/components/QA/Questions/AnswerModal.jsx (hunk 1 above)
Comment (hepnerthomas) [reply client/src/components/QA/Questions/AnswerModal.jsx]: think you commented on wrong lines of code. Here's what it looks like now: 
image

**Review by amberlyM — APPROVED** [review]

**Pull request merged**